lib/identity_cache/query_api.rb

```diff
@@ -121,6 +121,13 @@ def check_association_scope(association_name)
       end
 
       def record_from_coder(coder) #:nodoc:
+        return unless coder
+        ActiveSupport::Notifications.instrument('identity_cache.hydration', class: coder[:class]) do
+          record_from_coder_internal(coder)
+        end
+      end
+
+      def record_from_coder_internal(coder) #:nodoc:
```

Comment:
As already mentioned in https://github.com/Shopify/identity_cache/pull/340#discussion_r117493405, I don't like calling this internal, since it makes it sound like `record_from_coder` isn't internal